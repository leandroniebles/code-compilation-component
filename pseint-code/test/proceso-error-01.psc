imprimir "externo"
Proceso sin_titulo
FinProceso
imprimir "externo"
