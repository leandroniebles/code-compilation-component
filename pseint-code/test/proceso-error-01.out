=== Line 1: SynError 43
=== Line 4: SynError 43
